docs/build.ps1

```diff
@@ -0,0 +1,16 @@
+.\bonsai\modules\Export-Image.ps1 "..\src\Bonsai.ZeroMQ\bin\Release\net472"
+$exampleEnvironments = Get-ChildItem -Directory -Recurse -Filter ".bonsai" -Path "examples"
+foreach ($environment in $exampleEnvironments)
+{
+    # bootstrap environment
+    $galleryPath = Join-Path $environment.FullName "Gallery"
+    New-Item -Path $galleryPath -Force -ItemType Directory | Out-Null
+    Get-ChildItem -Path "..\src\bin\Release" -Filter *.nupkg | Copy-Item -Destination $galleryPath
+    . (Join-Path $environment.FullName "Setup.ps1")
+
+    # export example workflows
+    $workflowPath = $environment.Parent.FullName
+    $bootstrapperPath = Join-Path $environment.FullName "Bonsai.exe"
+    .\bonsai\modules\Export-Image.ps1 -workflowPath $workflowPath -bootstrapperPath $bootstrapperPath
+}
```

Comment:
```suggestion
```

The build script seems adapted from the machinelearning repo, but that repo is a bit of a special case since there are many complex examples each with their own environment folder requiring independent bootstrapping.

In the general case we should probably use a simpler build script such as the one we used for the [bonsai-rx/pulsepal](https://github.com/bonsai-rx/pulsepal/blob/main/docs/build.ps1) repo.

Reply:
fixed!